lua/wikis/commons/Widget/NavBox/Transfer.lua

```diff
@@ -86,26 +95,33 @@ end
 ---@param pagesByYear table<integer, string[]>
 ---@return Widget[]
 ---@return Widget[]
-function TransferNavBox._getUnsortedAndUnsourced(pagesByYear)
+---@return Widget[]
```

Comment:
IIRC the annotation supports naming multi returns,
e.g.
```suggestion
---@return Widget[] yearly
```
(Can't suggest this for all lines due to github)